rclpy/rclpy/__init__.py

```diff
@@ -35,8 +35,8 @@ def init(args=None):
     return _rclpy.rclpy_init(args if args is not None else sys.argv)
 
 
-def create_node(node_name):
-    node_handle = _rclpy.rclpy_create_node(node_name)
+def create_node(node_name, namespace=None):
```

Comment:
A namespace can be empty. A topic name can't be. Since both types have separate validation criteria I would suggest to create separate validation functions for them. These implementation could share common functionality though if that makes sense.